packages/composite-checkout/src/components/checkout-modal.js

```diff
@@ -2,7 +2,8 @@
  * External dependencies
  */
 import React, { useEffect } from 'react';
-import styled, { keyframes } from '@emotion/styled';
+import styled from '@emotion/styled';
+import { keyframes } from '@emotion/core';
```

Comment:
Oops. Sorry I missed this in my migration!

Reply:
No problem!